lib/u3d/installer.rb

```diff
@@ -311,7 +311,11 @@ def install_exe(file_path, installation_path: nil, info: {})
         command = nil
         if info['cmd']
           command = info['cmd']
-          command.sub!(/{FILENAME}/, file_path)
+          if /msiexec/ =~ command
+            command.sub!(/{FILENAME}/, '"' + file_path.gsub(%r{\/(\d)}, '/\\\\\1').tr('/', '\\') + '"')
```

Comment:
turn this into a private method?

windows_path(file_path) or something?

Reply:
Maybe put in utils?
This could be reusable

Reply:
as you wish. At least extract it from here to that we can understand what happens without decrypting the code :)